horace_core/sqw/axes_blocks/@AxesBlockBase/AxesBlockBase.m

```diff
@@ -719,14 +719,14 @@
             %       column describes a point on a grid.
             %
             % Inputs:
-            % obj         -- initialized version of the axes block
+            % obj        -- initialized version of the axes block
+            %
             % Optional:
-            %  char_cube -- the cube, describing the scale of the grid,
-            %               to construct the lattice on, defined by its
-            %               minimal and maximal points (4x2 matrix)
-            %  or         --
-            %               char_size directly (4x1 vector), describing the
-            %               scales along each axis the lattice should have
+            % nbins_all_dims 
+            %           -- (4-element vector) or single number
+            %               expanded to 4-element vector, describing the
+            %               binning along each axis for the lattice to
+            %               build instead of existing axis binning
```

Comment:
presuming this relates to the use of nbins_all_dims below, would be worth cross referencing to this comment at that point